src/java-github-release.sh

```diff
@@ -7,6 +7,26 @@ throw() {
   exit 1
 }
 
+get_changelog () {
```

Comment:
Doesn't this function need to output the change log? Otherwise calling it (`$(get_changelog)`) won't return anything.

Reply:
Should I use `echo` or `printf` to output `$changelog`

Reply:
:man_shrugging: iirc I used `printf` originally, but I don't remember why.

Reply:
Used `printf` to keep format changelog format